pre_commit/resources/hook-tmpl

```diff
@@ -40,6 +41,7 @@ fi
 {pre_push}
 
 # Run pre-commit
+args="$args --hook-stage $hook_stage"
```

Comment:
this is not quite right, it'll end up calling:

`pre-commit --hook-stage commit`

when it needs to call

`pre-commit run --hook-stage commit`